nixos/tests/predictable-interface-names.nix

```diff
@@ -0,0 +1,27 @@
+{ system ? builtins.currentSystem
+, pkgs ? import ../.. { inherit system; }
+}:
+with import ../lib/testing.nix { inherit system; };
+let boolToString = x: if x then "yes" else "no"; in
+let testWhenSetTo = predictable: withNetworkd:
+makeTest {
+  name = "predictableInterfaceNames-${boolToString predictable}-with-networkd-${boolToString withNetworkd}";
```

Comment:
I would like these attribute names to be more descriptive.
Instead of `yes` and `no` I think we should do something along these lines:

`"predictableInterfaceNames-${if predictable then "" else "un"}predictable${if withNetworkd then "-with-networkd" else ""}"`